docs/evm-sidechain/bridge-client.md

```diff
@@ -7,6 +7,9 @@ status: not_enabled
 ---
 # Bridge Client
 
-The bridge client provides a user-friendly interfact to interact with your bridge. You can use the client to connect wallets and transfer XRP or tokens across XRPL chains.
+The bridge client provides a user-friendly interface to interact with your bridge. You can use the client to connect wallets and transfer XRP or tokens across XRPL chains.
```

Comment:
"friendly" is subjective, of course, and this makes as much sense without it.

```suggestion
The bridge client provides a user interface to interact with your bridge. You can use the client to connect wallets and transfer XRP or tokens across XRPL chains.
```

Reply:
Also, by "your" bridge, you mean this is meant to be run by the entities _running_ the bridge? Or can this be used by "common" users to interact with a bridge run by someone else?